Create lwm2m-agent-1021-11-2-limited-number-of-events-produced-when-receiving-data-from-lwm-2.md (#3712)
SoftwareAG/c8y-docs: content/change-logs/device-management/lwm2m-agent-1021-11-2-limited-number-of-events-produced-when-receiving-data-from-lwm-2.md

---
date: ""
title: Limited number of events produced when receiving data from LWM2M device
product_area: Device management & connectivity
change_type:
  - value: change-VSkj2iV9m

>     label: Fix

Reviewer: ```suggestion
  - value: change-2c7RdTdXo4
    label: Improvement
```

component:
  - value: component-1KLUzmqfe
    label: LWM2M
build_artifact:
  - value: tc-ggH2M4hf3
    label: lwm2m-agent
ticket: DM-4797
version: 1021.11.2
---

> In several CD versions, the multi-instance LWM2M resources was being logged as separate events for each resource instance in the verbose logging level. This change simplifies the logs by combining all resource-instance values into a single event.

Reviewer 2: ```suggestion
Previously, each resource instance of multi-instance LWM2M resources was logged as a separate event in the verbose logging level. This change simplifies the logs by combining all resource-instance values into a single event.
```

> Description:  In the 2025 release, the LWM2M agent introduced configurable event log levels per device. As a result, data from multi-instance LWM2M resources was being logged as separate events for each instance. This fix simplifies the logs by combining all resource-instance values into a single event.

Reviewer: ```suggestion
In several CD versions, the multi-instance LWM2M resources was being logged as separate events for each resource instance in the verbose logging level. This change simplifies the logs by combining all resource-instance values into a single event.
```
